src/dotnet/Fable.Compiler/FSharp2Fable.fs

```diff
@@ -1122,7 +1238,7 @@ type FableCompiler(com: ICompiler, state: ICompilerState, checkedProject: FSharp
     member fcom.UsedVarNames = set usedVarNames
     interface IFableCompiler with
         member fcom.Transform ctx fsExpr =
-            transformExpr fcom ctx fsExpr
+            transformExpr [] fcom ctx [fsExpr]|> List.head
```

Comment:
iirc the places where this is used replaces instances where a single expr is all that was used and there's a one to one correspondence between input elements and output elements so with a one element list input `[fsExpr]` the output will also only be one element.

I also made a `transformOneExpr` helper function to simplify but I guess I didn't replace all the one input calls that I could have